peer/snapshot.proto

```diff
@@ -13,26 +13,35 @@ import "google/protobuf/empty.proto";
 
 // SnapshotRequest contains information for a generate/cancel snapshot request
 message SnapshotRequest {
-    string channel_id = 1;
-    uint64 height = 2;
+    // The identity of the client that is used to verify the policy and signature in SignedRequest.
+    // It is a msp.SerializedIdentity in bytes form
+    bytes client_identity = 1;
```

Comment:
What is the benefit of including this in the raw request of each type? Why not to have it in `SignedSnapshotRequest`?

Reply:
@manish-sethi Thank you for the comment. Per offline discussion, I have updated the proto to (1) include both identity and nonce (i.e., SignatureHeader) (2) cover signer's identity in signature.